perception/image_projection_based_fusion/docs/roi-detected-object-fusion.md

```diff
@@ -18,32 +21,34 @@ The `DetectedObject` has three shape and the polygon vertices of a object are as
 
 ### Output
 
-| Name                 | Type                                                  | Description                                       |
-| -------------------- | ----------------------------------------------------- | ------------------------------------------------- |
-| `output`             | `autoware_auto_perception_msgs::msg::DetectedObjects` | detected objects                                  |
-| `output/image_rawID` | `sensor_msgs::msg::Image`                             | images for visualization, `ID` is between 0 and 7 |
+| Name                      | Type                                                  | Description                |
+| ------------------------- | ----------------------------------------------------- | -------------------------- |
+| `output`                  | `autoware_auto_perception_msgs::msg::DetectedObjects` | detected objects           |
+| `~/debug/image_raw[0-7]`  | `sensor_msgs::msg::Image`                             | images for visualization,  |
+| `~/debug/fused_objects`   | `autoware_auto_perception_msgs::msg::DetectedObjects` | fused detected objects     |
+| `~/debug/ignored_objects` | `autoware_auto_perception_msgs::msg::DetectedObjects` | not fused detected objects |
 
 ## Parameters
 
 ### Core Parameters
 
-| Name            | Type  | Description                                                                    |
-| --------------- | ----- | ------------------------------------------------------------------------------ |
-| `use_iou_x`     | bool  | calculate IoU only along x-axis                                                |
-| `use_iou_y`     | bool  | calculate IoU only along y-axis                                                |
-| `use_iou`       | bool  | calculate IoU both along x-axis and y-axis                                     |
-| `iou_threshold` | float | the IoU threshold to overwrite a label of a detected object with that of a roi |
-| `rois_number`   | int   | the number of input rois                                                       |
-| `debug_mode`    | bool  | If `true`, subscribe and publish images for visualization.                     |
+| Name                                            | Type   | Description                                                                                                  |
+| ----------------------------------------------- | ------ | ------------------------------------------------------------------------------------------------------------ |
+| `rois_number`                                   | int    | the number of input rois                                                                                     |
+| `debug_mode`                                    | bool   | If `true`, subscribe and publish images for visualization.                                                   |
```

Comment:
If set to `true`, the node subscribes to the image topic and publishes an image with debug drawings.

Reply:
updated at [886ce8a](https://github.com/autowarefoundation/autoware.universe/pull/2178/commits/886ce8a894c29e8fd0df34f56bc743980583a86b).